configs/vm_resource.py

```diff
@@ -29,6 +29,14 @@ class Project(enum.Enum):
   TPU_PROD_ENV_AUTOMATED = "tpu-prod-env-automated"
 
 
+class ImageProject(enum.Enum):
+  DEEP_LEARNING_PLATFORM_RELEASE = "deeplearning-platform-release"
```

Comment:
My concern is that if in the future we use VM that doesn't require the reinstallation of the drivers, removing he image may cause issue in that case. At this time I added `--force` in `sudo python3 install_gpu_driver.py --force` to force the installation.

I think we can keep it here since it won't do any harm in case driver is not reinstalled in the future. WDYT?